app/components/catalog-settings/component.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  actions: {
+    keymapChanged: function() {
+      this.sendAction('keymapChanged', this.get('keymap.asMap'));
+    },
+  },
+
+  keymap: Ember.Object.create({}),
+
+  didInitAttrs: function() {
+
+    var catalogs = this.get('catalog.value');
+    var catalogsSplit = catalogs.split(',');
+    var keymapOut = {};
+
+    if (catalogsSplit.length > 1) {
+
+      // its not default so we need to parse it
+      catalogsSplit.forEach((item) => {
+        var split = item.split('=');
+        keymapOut[split[0]] = split[1];
```

Comment:
Trim both parts to be tolerant of user inputs with spaces around the `=` and `,`. (also below)